container/features/pom.xml

```diff
@@ -232,7 +232,7 @@
             <feature>opennms-elasticsearch-event-forwarder</feature>
 
             <!-- BSM -->
-            <feature>org.opennms.features.bsm.service</feature>
+            <feature>org.opennms.features.bsm.service.api</feature>
```

Comment:
I am not 100% sure how the feature and repository section here is analyzied. But I think we need to add a dependency to org.opennms.features.bsm.service.api in the dependencies section of the pom to ensure that the projects are build in the right order.